src/commands/scratch-create.yml

```diff
@@ -0,0 +1,22 @@
+# Creates a scratch SFDX Org.
+description: >
+  'Create a scratch Salesforce Org.'
+parameters:
+  scratch-config:
+    description: 'The scratch JSON definition.'
+    type: string
+  scratch-alias:
+    description: 'The alias of the scratch Org.'
+    type: string
+  overrides:
+    description: 'Overrides for scratch organization.'
+    type: string
+    default: ''
+
+steps:
+  - run:
+      alias: Create << parameters.scratch-alias >> SalesForce Org
```

Comment:
Alias? Should this be "name"? https://circleci.com/docs/2.0/configuration-reference/#run

Reply:
I debated this as well - alias is how it's referred to in the CLI, and I figured it would be best to align with that naming.